### ISSUE TITLE
PanAz: ar position arg is scaled wrongly

### ISSUE THREAD

**Issue by dkmayer** [supercollider/supercollider#3148]
http://new-supercollider-mailing-lists-forums-use-these.2681727.n2.nabble.com/PanAz-bug-ar-position-arg-is-scaled-wrongly-td7634788.html

simple reproducer:

```
// ch 0
{ Out.ar(0, PanAz.ar(8, Saw.ar(400, 0.1), DC.kr(0), orientation: 0)) }.scope

// ch 1
{ Out.ar(0, PanAz.ar(8, Saw.ar(400, 0.1), DC.ar(0), orientation: 0)) }.scope
```

**Comment by Sciss** [comment]
Yes, PanAz pos must be control-rate: https://github.com/Sciss/ScalaColliderUGens/issues/30

**Comment by dkmayer** [comment]
But what do you mean by 'must be' ?
If you scale it differently PanAz works in principle (as shown in the post), or does it perform an A2K interpolation ?
Also PanAz.ar is used in helpfile examples ...

**Comment by Sciss** [comment]
By 'must be' I mean I can confirm it is broken for audio-rate - at least when I conducted my tests; perhaps with certain inputs it works, but I don't think it works in principle... Probably current version should I either refuse audio rate input or the UGen needs to be fixes?

**Comment by dkmayer** [comment]
A fix would be highly appreciated. There are fast movements where ar pos args definitely make a difference (often noticed with other Panners).
BTW above I wrote
> PanAz.ar ...
of course I meant 'PanAz with ar pos arg'.

**Comment by LFSaw** [comment]
If one coffee is enough, I think I found and fixed the culprit (see #3150). please test and report back there.

**Comment by dkmayer** [comment]
Great, thanks for looking into that !
I'll test in the evening.

**Comment by LFSaw** [comment]
fixed with #3150
